src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -989,19 +1059,18 @@ public override int GetHashCode()
 
             if (_bits == null)
                 return _sign;
-            int hash = _sign;
-            for (int iv = _bits.Length; --iv >= 0;)
-                hash = NumericsHelpers.CombineHash(hash, unchecked((int)_bits[iv]));
-            return hash;
+
+            HashCode hash = default;
+            hash.Add(_sign);
+            hash.AddBytes(MemoryMarshal.AsBytes(_bits.AsSpan()));
```

Comment:
Please revert the changes to this method. `MemoryMarshal.AsBytes` could throw _OverflowException_. This could break people who are creating extraordinarily large `BigInteger` values. (Why would anybody ever do this? Who knows...)

Reply:
Done with small improvement - `CombineHash` is now local static function